API/functions/callContract.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
```

Comment:
I believe this is untested, I'd like to see some solid tests to ensure there is no bugs when calling a contract. This will be exposed to the world and we have to assume that it will get hit with all kinds of rubbish.

Maybe move this onto another branch until its tested